x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/summary_step/summary.tsx

```diff
@@ -11,29 +11,36 @@ import { toastNotifications } from 'ui/notify';
 import { WizardNav } from '../wizard_nav';
 import { WIZARD_STEPS, StepProps } from '../step_types';
 import { JobCreatorContext } from '../job_creator_context';
+import { JobRunner } from '../../../common/job_runner';
 import { mlJobService } from '../../../../../services/job_service';
 import { JsonFlyout } from './json_flyout';
 import { isSingleMetricJobCreator } from '../../../common/job_creator';
 import { JobDetails } from './job_details';
 import { DetectorChart } from './detector_chart';
 import { JobProgress } from './components/job_progress';
+import { PostSaveOptions } from './components/post_save_options';
 
+// ts-ignore
```

Comment:
Can you add a comment why this is needed (maybe a TODO about what needs to be done to be able to remove it)? (Does this even trigger without `@`?)

Reply:
accidentally left in from a js include that needed it but was then removed.
good spot!